roles/rhcos-post-install/tasks/main.yml

```diff
@@ -1,7 +1,7 @@
 ---
 - name: Get cluster name
   shell: |
-    {%raw%}oc get machineset -n openshift-machine-api -o=go-template='{{(index (index .items 0).metadata.labels "machine.openshift.io/cluster-api-cluster")}}'{%endraw%}
+    {%raw%}oc get machineset -n openshift-machine-api -o=go-template='{{(index (index .items 0).metadata.labels "sigs.k8s.io/cluster-api-cluster")}}'{%endraw%}
```

Comment:
I think this was an unintended change.

Reply:
@akrzos will open an issue to track the labels difference in the latest installer/image.